Source/src/modules/ModuleResources.h

```diff
@@ -30,7 +30,7 @@ namespace Hachiko
         void HandleResource(const std::filesystem::path& path);
 
     private:
-        std::map<UID, Resource*> loaded_resources;
+        std::map<UID, Resource*> loaded_resources{};
```

Comment:
I believe list-initialization requires at least one element, hence this would not be that. This comes from a tool suggestion to make sure the collections are properly initialized, for vector here's something I found:
`The default constructor of std::vector constructs an empty container with a default-constructed allocator, meaning – initialize empty vector C++. Alternatively, you can pass a specific allocator as the only function argument to utilize it. The default constructor is called when a variable is declared without an initializer. Notice that if you specify empty parentheses after the variable name, it will create a function and not a default-constructed vector.`